CHANGELOG.md

```diff
@@ -1135,6 +1135,7 @@ activity and on its child transactions (which can be actuals, refunds, and adjus
 - Do not allow removing the implementing organisation from a level C or D ISPF activity if it is the only one
 - Add ISPF Level C/D UI user journeys for adding activities
 - Remove collaboration type and FSTC applies from UI user journeys for adding Level B ISPF activities
+- Update ISPF Level B UI user journey for adding activities
```

Comment:
I suppose more specifically this could be: "Remove non-ODA steps from new ISPF Level B Activity form"?